src/Boolean.php

```diff
@@ -4,31 +4,30 @@
 
 use InvalidArgumentException;
```

Comment:
I didn't look at that part, but I figured the tests would break it was invalid. I don't think any of these `use InvalidArgumentException` statements are needed then, unless you also want to add a `@throws` annotation on the various constructors